Project.toml

```diff
@@ -22,13 +21,12 @@ SymbolicIndexingInterface = "2efcf032-c050-4f8e-a9bb-153293bab1f5"
 [compat]
 FastBroadcast = "0.3.5"
 LinearAlgebra = "1"
-LinearSolve = "2.32"
+LinearSolve = "2.32, 3"
```

Comment:
Was the new alias infrastructure introduced in LinearSolve.jl v3? If so, we cannot support the old version with this PR anymore (which is fine).

Reply:
Do you mean the `LinearAliasSpecifier`? This is defined in SciMLBase.jl and was introduced in v2.68.0. So this is covered by our current compat bounds. Would still be interesting if we can make Downgrade happy with LinearSolve.jl v2.32. I will work on the Downgrade action to check that.

Reply:
I think we need to bump the compat to v3.7 for downgrade anyway.